sormas-ui/src/main/java/de/symeda/sormas/ui/utils/AbstractEditForm.java

```diff
@@ -240,6 +240,29 @@ protected ComboBox addInfrastructureField(String fieldId) {
 		field.setWidth(100, Unit.PERCENTAGE);
 	}
 
+	@SuppressWarnings("rawtypes")
+	@Override
+	protected <T extends Field> void formatField(T field, String propertyId, String customCaption) {
```

Comment:
A lot of duplicated code; this method can stay as it is, but the already existing formatField method should be changed. Basically remove everything but the line that builds the caption and then use that to call the new method:

```
formatField(field, propertyId, I18nProperties.getPrefixCaption(propertyI18nPrefix, propertyId, field.getCaption()));
```